src/main/java/xbot/common/controls/actuators/XCANMotorController.java

```diff
@@ -223,33 +235,212 @@ public void periodic() {
 
     public abstract void setPowerRange(double minPower, double maxPower);
 
-    public Angle getPosition() {
+    public void setDistancePerAngleScaleFactor(Measure<? extends PerUnit<DistanceUnit, AngleUnit>> distancePerAngle) {
```

Comment:
This pair of methods is definitely going to need some javadoc to explain to students why they might want to call this and what the input arguments need to be. Would also suggest including a specific example, like an elevator an/or arm for each function to hammer that home.

Reply:
Added!